<?xml version="1.0"?>
<launch>

  <node name="tubaf_haf_client" pkg="tbf_gripper_autonomy" type="haf_client">
    <rosparam file="$(find tbf_gripper_autonomy)/config/std_params.yaml" command="load" />
  </node>
</launch>